### DISCUSSION TITLE
Multiple recordings in one meeting

### DISCUSSION THREAD

**Discussion by freelimiter** [hyunbinseo/zoom-rec-dl#10] [Q&A] [answered]
Hello, first of all thank you for creating this! It really saved me some time downloading zoom recordings.

We have zoom recordings that have multiple video files.  Currently, your code only downloads the first video. 
Could you please make it so that it downloads all video files? 

Thank you!

**Comment by hyunbinseo** [comment]
Could you list all downloaded filenames and the one that you need?

**Reply by freelimiter** [reply]
can I message you or email you privately? I can give you the actual link that has multiple videos in it. Thanks

**Reply by hyunbinseo** [reply]
Would be nice if you could create a demo URL that can be shared publicly.

Based on my experiences and environment, there are three types of video files.

- Recording_gvo_1280x720
- Recording_as_640x360
- Recording_avo_640x360

Are you trying to download one of the above?

**Reply by freelimiter** [reply]
We run webinars/meetings that is more than 5 hours. And zoom splits them up into several video files.
Unfortunately I can not share the link due to company proprietary information. But here's the screenshot of the shared link:
At the bottom of the video player, the |> button or forward button plays the Recording 2 of the video. In this example, there are 2 video in this link. There could be more in meetings.

![image](https://user-images.githubusercontent.com/1796267/223745572-21a0b1c6-e794-4168-b437-d63fec1d3d8c.png)

**Reply by hyunbinseo** [reply]
I guess I have to see the URL for myself. Send an email to 59DC15947A4CA@altmails.com

**Reply by hyunbinseo** [reply]
I have received the email. Would take some time to implement, since the new download is in another URL.

**Comment by hyunbinseo** [comment, marked as answer]
Long Zoom recordings with multiple parts are now supported. https://github.com/hyunbinseo/zoom-rec-dl/commit/608c5fa63a239a1f85713b8d8ad11d5ff7dadac8

Please re-try using the latest script.
